Runtime/CesiumPointCloudRenderer.cs

```diff
@@ -17,6 +17,7 @@ internal struct Cesium3DTileInfo
     }
 
     [ExecuteInEditMode]
+    [AddComponentMenu("Cesium/Cesium Point Cloud Renderer")]
```

Comment:
This component is internal and meant only to be added by tilesets. I'm surprised it shows up in the component menu despite its C# class visibility.

Giving the attribute an empty string hides the component from the list. Let's do this here to hide this from users.

```suggestion
    [AddComponentMenu("")]
```